### ISSUE TITLE
Post Checkout: Premium plan upsell needs copy review

### ISSUE THREAD

**Issue by sixhours** [Automattic/wp-calypso#36056]
Upon checking out with a Personal plan, you get the following upsell:

<img width="996" alt="screencapture-wordpress-checkout-calobeetesting06-food-blog-offer-plan-upgrade-premium-40249038-2019-09-09-15_20_35 (1)" src="https://user-images.githubusercontent.com/2124984/64563352-23d22e00-d31d-11e9-9e27-895014a9bc22.png">

This page could use an in-depth editorial copy review; I have a hunch most people don't read this much, and the page could perform better if we made it more concise.

Until then, there are some typos/grammatical errors in the copy that can be addressed, and a couple places where we could tighten things up:

1) Heading: Underlined text usually represents a link, but not in this case. We could find a different way to emphasize these bits to avoid confusion if it's absolutely necessary; italics? Asterisks?
2) Typo in the 6th paragraph; "looks" should be "look"
3) Typo in the 9th paragraph, “cost” should be “costs”. 
4) 9th paragraph: Is there a reason we specify “WordPress themes”? They're already on WordPress.com. I don't think they're looking for Blogger or Tumblr themes... :D 
5) 10th paragraph: "...just an additional $36" is confusing, after having referenced the cost of the individual themes in the previous paragraph. $36 in addition to the cost of the individual themes? Or the plan I've already purchased? We could probably remove the "an additional" from this sentence to simplify and make it clearer.
6) 2nd bullet point beginning "Advanced tools...": I think the “to” is meant to be an “or” in this sentence? “Schedule posts in advance, resurface your older content, **or** schedule multiple social posts at a time.”
7) 3rd bullet point beginning "Customize...": Missing word, “stand” should be “stand out”.
8) 3rd bullet point beginning "Customize...": “and never be the same as others” is redundant; we could remove that bit to simplify this sentence.
9) In the last line, why is the bit about themes in parentheses? It seems like the themes are the important thing we're trying to sell on this page, so the parentheses should probably be removed.
10) We refer to both "themes" and "designs" on this page, using the terms interchangeably. We should be consistent to avoid confusion. Change "designs" to "themes" to match the wording across the rest of WordPress.com.
11) "Designs" is capitalized in the page's heading, but lowercase in the page's primary action button at the bottom. It should probably be lowercase in both instances.
12) For the skip button, "No thanks" would probably suffice.

**Comment by southp** [comment]
@niranjan-uma-shankar I sort of remember that you were involved in implementing this upsell message so I'm cc-ing you here :)

Also, @michelleweber could you share your insights on this?

**Comment by donlair** [comment]
We're always happy to run a test if you write different copy to throw into the mix. This is a brand new offer, and this is the first approach we've tested, so it's quite possible that the shorter version will win. Once you have the copy ready, we can slot the test in our queue.

We ran a similar short vs long test with the Quick Start session offer and the longer version's conversion rate was 2x the short version, so we went a longer version here for our first iteration.

We'll take look at the grammatical issues you highlighted and correct as needed, thanks for raising them. CC @victorespigares
